traffic_ops/app/lib/UI/DeliveryService.pm

```diff
@@ -152,12 +153,7 @@ sub get_example_urls {
 				}
 			}
 			elsif ( $re->{type} eq 'PATH_REGEXP' ) {
-				if ( defined( $example_urls[ $re->{set_number} ] ) ) {
-					$example_urls[ $re->{set_number} ] .= $re->{pattern};
-				}
-				else {
-					$example_urls[ $re->{set_number} ] = $re->{pattern};
-				}
+				push(@example_urls, $re->{pattern});
```

Comment:
What made this change necessary? Doesn't the old code concatenate patterns if they use the same `set_number`? This looks like it would just add multiple different patterns using the same `set_number`s.